InfluxData.Net.InfluxDb/ResponseParsers/UserResponseParser.cs

```diff
@@ -12,7 +12,7 @@ public virtual IEnumerable<User> GetUsers(IEnumerable<Serie> series)
         {
             var users = new List<User>();
 
-            if (series == null || series.Count() == 0)
+            if (series == null || series.Count() == 0 || series.Single().Values == null)
```

Comment:
Well, it has been a little bit of time now so part of this is from memory, but yes I think it should be fine. It definitely worked in all my testing. As far as I know there is ever only one series which is the single list of users and the if the count() == 0 then it won't get to the next || check there. But you are right, it might not hurt to switch to First() for future proofing. If series.Count() > 0 then there is definitely going to be at least First() so FirstOrDefault() isn't necessary or even a good choice. I vote First() to be safe.